conf/grub.cfg

```diff
@@ -3,3 +3,5 @@
 #   set_global eve_flavor kvm
 # to force booting in Xen mode, uncomment:
 #   set_global eve_flavor xen
+set_global hv_dom0_mem_settings "dom0_mem=2G,max:2G"
```

Comment:
@rouming , changes here won't go to the rootfs... you need to change pkg/grub/rootfs.cfg in order to get the changes injected into the rootfs...